config/sources/families/mvebu64.conf

```diff
@@ -111,11 +111,12 @@ uboot_custom_postprocess()
 		cp build/a80x0_mcbin/release/flash-image.bin $ubootdir/$FILENAME
 	else
 	    cd $moxbootdir
+    	make clean
     	make CROSS_CM3=$ATF2 wtmi_app.bin
 	    cd $atfdir
-	
+
 		clocks=( 600_600 800_800 1000_800 1200_750 )
-		topology=( 512m_1cs_0 512m_2cs_0 1g_2cs_2 1g_1cs_4 2g_2cs_7 1g_1cs_5 2g_2cs_6 )
```

Comment:
Why this fell out?

Reply:
From https://trustedfirmware-a.readthedocs.io/en/latest/plat/marvell/armada/build.html, the second time the heading DDR Topology is on the page, it gives a list of "supported options".  Topology zero is a single 512 chip of DDR3, topology 1 is a single chip of DDR4.  There is no 2 chip 512 topology. I didn't add topology 1 for the same reason we don't build topology 3 - it doesn't apply to a supported board.  We don't support Armada-3700 DB